tests/Reporter/GitlabReporterTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfTwig\Twigcs\Tests\Reporter;
+
+use FriendsOfTwig\Twigcs\Reporter\GitlabReporter;
+use FriendsOfTwig\Twigcs\Validator\Violation;
+use JsonException;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+class GitlabReporterTest extends TestCase
```

Comment:
@frankdekker 

Can this class be `final`?

Reply:
done